Formula/fasthenry-wr.rb

```diff
@@ -0,0 +1,34 @@
+class FasthenryWr < Formula
+  desc "3D inductance computation program which handles superconducting conductors"
+  homepage "http://www.wrcad.com/freestuff.html"
+  url "http://www.wrcad.com/ftp/pub/fasthenry-3.0wr-071720.tar.gz"
+  version "3.0-071720"
```

Comment:
How does one know the correct version based on the tarball `url`? Do you always just remove the letters in between the digits?

Reply:
"wr" refers to the fact that this version of fasthenry is a fork from Whiteley Research Inc.

Reply:
Ok, but that answers neither of my questions.

Reply:
Apologies for my oversight.

The tarball has a version number and a date, and both of these make up the version number. It looks like version number is always 3.0, but the date changes. 

(An argument could be made that this package should be called `fasthenry3`, but I haven't seen any evidence that anyone else uses this number).

The letters in between the digits can and should be removed. They just indicate that its the wrcard fork of `fasthenry`, which seems to be the most popular fork.